cpufreq/src/cpufreq-monitor.c

```diff
@@ -249,150 +249,150 @@ cpufreq_monitor_get_property (GObject    *object,
+        break;
+    case PROP_FREQUENCY:
+        g_value_set_int (value, monitor->priv->cur_freq);
+        break;
+    case PROP_MAX_FREQUENCY:
+        g_value_set_int (value, monitor->priv->max_freq);
+        break;
+    case PROP_GOVERNOR:
+        g_value_set_string (value, monitor->priv->governor);
+        break;
+    default:
+        G_OBJECT_WARN_INVALID_PROPERTY_ID (object, prop_id, spec);
+        break;
+    }
 }
 
 static gboolean
 cpufreq_monitor_run_cb (CPUFreqMonitor *monitor)
 {
-        CPUFreqMonitorClass *class;
-        gboolean             retval = FALSE;
+    CPUFreqMonitorClass *class;
+    gboolean             retval = FALSE;
 
-        class = CPUFREQ_MONITOR_GET_CLASS (monitor);
-        
-        if (class->run)
-                retval = class->run (monitor);
+    class = CPUFREQ_MONITOR_GET_CLASS (monitor);
 
-        if (monitor->priv->changed) {
-                g_signal_emit (monitor, signals[SIGNAL_CHANGED], 0);
-                monitor->priv->changed = FALSE;
-        }
+    if (class->run)
+        retval = class->run (monitor);
+
+    if (monitor->priv->changed) {
+        g_signal_emit (monitor, signals[SIGNAL_CHANGED], 0);
+        monitor->priv->changed = FALSE;
+    }
 
-        return retval;
+    return retval;
 }
 
 void
 cpufreq_monitor_run (CPUFreqMonitor *monitor)
 {
-        g_return_if_fail (CPUFREQ_IS_MONITOR (monitor));
+    g_return_if_fail (CPUFREQ_IS_MONITOR (monitor));
 
-        if (monitor->priv->timeout_handler > 0)
-                return;
+    if (monitor->priv->timeout_handler > 0)
+        return;
 
-        monitor->priv->timeout_handler =
-                g_timeout_add_seconds (CPUFREQ_MONITOR_INTERVAL,
+    monitor->priv->timeout_handler =
+        g_timeout_add_seconds (CPUFREQ_MONITOR_INTERVAL,
                                (GSourceFunc) cpufreq_monitor_run_cb,
                                (gpointer) monitor);
 }
 
 GList *
 cpufreq_monitor_get_available_frequencies (CPUFreqMonitor *monitor)
 {
-        CPUFreqMonitorClass *class;
-        
-        g_return_val_if_fail (CPUFREQ_IS_MONITOR (monitor), NULL);
-
-        if (!monitor->priv->online)
-                return NULL;
-        
-        if (monitor->priv->available_freqs)
-                return monitor->priv->available_freqs;
-
-        class = CPUFREQ_MONITOR_GET_CLASS (monitor);
-        
-        if (class->get_available_frequencies) {
-                monitor->priv->available_freqs = class->get_available_frequencies (monitor);
-        }
+    CPUFreqMonitorClass *class;
+
+    g_return_val_if_fail (CPUFREQ_IS_MONITOR (monitor), NULL);
 
+    if (!monitor->priv->online)
+        return NULL;
+
+    if (monitor->priv->available_freqs)
         return monitor->priv->available_freqs;
+
+    class = CPUFREQ_MONITOR_GET_CLASS (monitor);
+
+    if (class->get_available_frequencies) {
+        monitor->priv->available_freqs = class->get_available_frequencies (monitor);
+    }
```

Comment:
If there is only one line in the if-body it's not necessary to use the curly brackets